olm-catalog/serverless-operator-index/Dockerfile

```diff
@@ -21,12 +21,22 @@ registry.ci.openshift.org/knative/release-1.34.0:serverless-bundle \
 # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
 FROM registry.ci.openshift.org/origin/4.18:operator-registry
```

Comment:
This whole Dockerfile is only for test purposes. Konflux uses those under serverless-operator-index/vX.YY/Dockerfile